src/components/icon/icon.tsx

```diff
@@ -25,7 +25,7 @@ export class Icon {
   @State() svgHTML = '';
 
   private async getSVGHTML(iconName: string) {
-    const response = await fetch(getAssetPath(`assets/icons/${iconName}.svg`));
+    const response = await fetch(getAssetPath(`icon-assets/icons/${iconName}.svg`));
```

Comment:
Why the rename?

Reply:
there's an existing issue that we can't use 'assets' directory from more than one component. This is the suggested solution to the issue